docs/usage/configuration/config_documentation.md

```diff
@@ -3170,6 +3170,10 @@ Options for each entry include:
 * `client_secret`: oauth2 client secret to use. May be omitted if
   `client_secret_jwt_key` is given, or if `client_auth_method` is 'none'.
 
+* `client_secret_path`: path to the oauth2 client secret to use. With that
+   it's not necessary to leak secrets into the config file itself.
+   Mutually exclusive with `client_secret`.
```

Comment:
How does this interact with `client_secret_jwt_key`?

Reply:
Is there much to say except what was mentioned above? I.e. it's an override for `client_secret`, i.e. can be omitted if `client_secret_jwt_key` is given.

Reply:
If I didn't know the inner workings of this I think it would be unclear. It probably makes sense to just re-iterate the above.